src/jdk.javadoc/share/classes/jdk/javadoc/doclet/Reporter.java

```diff
@@ -80,6 +84,33 @@
      */
     void print(Diagnostic.Kind kind, DocTreePath path, String message);
 
+    /**
+     * Prints a diagnostic message related to a position within a range of characters in a tree node.
+     *
+     * Only kinds of {@code DocTree} that wrap a simple string value are supported as leaf nodes
+     * of the given path. This currently includes
+     * {@link CommentTree}, {@link DocTypeTree}, {@link ReferenceTree}, and {@link TextTree}.
+     *
+     * The positions are all 0-based character offsets from the beginning of string.
+     * The positions should satisfy the relation {@code start <= pos <= end}.
+     *
+     * @implNote
```

Comment:
Should be `@implSpec` instead; please make sure the CSR is updated as well since it also uses `@implNote`.